packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -114,11 +123,33 @@ function readContext<T>(context: ReactContext<T>): T {
   return context._currentValue;
 }
 
-function use<T>(): T {
-  // TODO: What should this do if it receives an unresolved promise?
-  throw new Error(
-    'Support for `use` not yet implemented in react-debug-tools.',
-  );
+function use<T>(usable: Usable<T>): T {
+  if (usable !== null && typeof usable === 'object') {
+    // $FlowFixMe[method-unbinding]
+    if (typeof usable.then === 'function') {
+      // TODO: What should this do if it receives an unresolved promise?
+      throw new Error(
+        'Support for `use(Promise)` not yet implemented in react-debug-tools.',
+      );
```

Comment:
Because we patch Dispatcher with these hooks, we can't really return something other than `useThenable(usable)` here.

We could display the user some string like `Promise<rejected | resolved>`, but don't support it atm.

Reply:
Should/can we create a non-fiber fork of `useThenable`?

Reply:
> Should/can we create a non-fiber fork of `useThenable`?

Looking at its implementation, I don't think we can, it relies on local counter `thenableIndexCounter`.
cc @acdlite

Reply:
Maybe https://github.com/facebook/react/pull/28233 is enough?